lib/naughty/core.lua

```diff
@@ -66,11 +66,19 @@ notifications, e.g.
 naughty.config = {
     padding = dpi(4),
     spacing = dpi(1),
-    icon_dirs = { "/usr/share/pixmaps/", },
+    icon_dirs = { "/usr/share/icons/hicolor/", "/usr/share/icons/", "/usr/share/pixmaps/", },
     icon_formats = { "png", "gif" },
     notify_callback = nil,
 }
 
+if beautiful.icon_theme then
+    local theme = beautiful.icon_theme
+    local theme_dir = "/usr/share/icons/" .. theme .. "/"
+    table.insert(naughty.config.icon_dirs, 1, theme_dir)
+    table.insert(naughty.config.icon_dirs, 1, theme_dir .. "base/")
+    table.insert(naughty.config.icon_dirs, 1, theme_dir .. "symbolic/")
+end
```

Comment:
Duh, my bad.

I was wondering about that, actually, but I was stupid enough to only test on my configuration. I'm guessing the best approach here is to have `beautiful.init` update `naughty.config`.